files/en-us/web/api/mediadevices/getdisplaymedia/index.html

```diff
@@ -55,7 +55,7 @@ <h3 id="Return_value">Return value</h3>
 <div class="notecard note">
   <p><strong>Note:</strong> Browser support for audio tracks varies, both in terms of
     whether or not they're supported at all by the media recorder and in terms of the
-    which audio source or sourcoes are supported. Check the {{anch("Browser
+    which audio source or sources are supported. Check the {{anch("Browser
```

Comment:
Thanks for the typo, but I  think the sentence is not grammarly correct.

What about:
```suggestion
     audio sources supported. Check the {{anch("Browser
```
?

Reply:
Funny what my eyes miss :)